docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,117 @@
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in the data center.
+Useful network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters can be deployed on provider networks, so the installer won't need to create
+tenant networks nor floating IPs.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Common prerequisites
+
+* [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* Neutron should be configured with the [port_security][4] extension so the installer can
+  add the allowed_address_pairs attribute to ports.
+* The Provider Network has to be created by the administrator with specific configuration (see the next section).
+* The Metadata IP (169.254.269.254) has to be routed for the provider network.
+
+
+## Considerations when creating provider networks
+
+* The provider network has to be owned by the project ID used to bootstrap the OCP cluster.
+
+    It is important that the provider network and subnet are owned by the same project that will be used
+    to install OpenShift (from the clouds.yaml) and we don't want them to be owned by `admin` otherwise
+    it'll cause Terraform to fail creating the ports.
+
+    Example where our project name is `openshift`:
+
+        openstack network create --project openshift (...)
+        openstack subnet create --project openshift (...)
+
+    More information can be found about how to create provider networks [here][5].
+
+* The Metadata IP has to be routed so the bootstrap nodes can fetch the required information
+  to provision the node and the cluster.
+
+  This can be achieved by specifying a route when creating the subnet:
+
+        openstack subnet create --host-route destination=169.254.169.254/32,gateway=$ROUTER_IP" (...)
+
+  (where the gateway is the IP of the router in charge of routing the traffic of the Provider Network).
+
+**Note:** This should not apply to OVN because Metadata is completely distributed across OpenStack compute nodes
+          and served from the ovnmeta-xxxxx-xxxx namespace. Therefore, there is nothing to do.
+
+* When deployment baremetal workers on provider networks with OVN, the subnet should be created with DHCP enabled
+  and the Neutron DHCP agent should be deployed and running. There is an [ongoing effort][6] to remove that
+  dependency.
+
+        openstack subnet create --dhcp
+
+
+## Deploying cluster with primary interface on a provider network with IPI
+
+
+- Requirements: make sure all requirements documented previously have been met.
+
+- Create install-config.yaml:
+
+    - Set `platform.openstack.apiVIP` to the IP address for the API VIP.
```

Comment:
We should perhaps say somewhere that `platform.openstack.apiVIP` and  `platform.openstack.ingressVIP` both need to be an unnassigned IP address on the `networking.machineNetwork.cidr`.